libs/dh/globalization/assets-localization/src/assets/i18n/en.json

```diff
@@ -489,8 +489,13 @@
       "editGridArea": "Edit",
       "noGridAreas": "No gridareas found",
       "columns": {
-        "gridArea": "Grid area",
-        "gridAreaCode": "Code"
+        "name": "Grid area",
+        "code": "Code",
+        "actorName": "english_translation",
+        "actorNumber": "english_translation",
```

Comment:
question: I assume the translations will be part of a follow-up PR?

Reply:
jep :)